vendor/assets/javascripts/fixedsticky.js

```diff
@@ -0,0 +1,191 @@
+;(function( win, $ ) {
+
+  function featureTest( property, value, noPrefixes ) {
+    // Thanks Modernizr! https://github.com/phistuck/Modernizr/commit/3fb7217f5f8274e2f11fe6cfeda7cfaf9948a1f5
+    var prop = property + ':',
+      el = document.createElement( 'test' ),
+      mStyle = el.style;
+
+    if( !noPrefixes ) {
+      mStyle.cssText = prop + [ '-webkit-', '-moz-', '-ms-', '-o-', '' ].join( value + ';' + prop ) + value + ';';
```

Comment:
This is a library that I grabbed. It does look a little off. I'm used to the `-webkit-property: value` syntax but it seems to work, so maybe there is something that I'm missing.